logical_switch.go

```diff
@@ -271,3 +271,57 @@ func (odbi *ovnDBImp) lslblistImp(lswitch string) ([]*LoadBalancer, error) {
 	}
 	return listLB, nil
 }
+
+func (odbi *ovnDBImp) lsExtIdsAddImp(lswitch string, external_ids map[string]string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+	row := make(OVNRow)
+	row["name"] = lswitch
+	lsuuid := odbi.getRowUUID(tableLogicalSwitch, row)
+	if len(lsuuid) == 0 {
+		return nil, ErrorNotFound
+	}
+	if external_ids == nil {
```

Comment:
how about the map is empty?

Reply:
if empty, it means we raise error because we are trying to add/del empty ext_id maps for a lswitch

Reply:
@softwarejl Handled the empty external_id and just returning in case the its null instead of raising error. Also covered the UT for the same.